docs/getting-started/faq.md

```diff
@@ -84,15 +88,15 @@ Yes. Specifically:
 
 ### Does Feast support X storage engine?
 
-The list of supported offline and online stores can be found [here](../reference/offline-stores/) and [here](../reference/online-stores/), respectively. The [roadmap](../roadmap.md) indicates the stores for which we are planning to add support. Finally, our Provider abstraction is built to be extensible, so you can plug in your own implementations of offline and online stores. Please see more details about custom providers [here](../how-to-guides/creating-a-custom-provider.md).
+The list of supported offline and online stores can be found [here](../reference/offline-stores/) and [here](../reference/online-stores/), respectively. The [roadmap](../roadmap.md) indicates the stores for which we are planning to add support. Finally, our Provider abstraction is built to be extensible, so you can plug in your own implementations of offline and online stores. Please see more details about custom providers [here](broken-reference).
```

Comment:
why did u link broken-reference here? I think the custom-provider reference works fine(I just checked it out in gitbook ../how-to-guides/customizing-feast/creating-a-custom-provider.md

Reply:
fixed!